src/sql/workbench/services/restore/common/constants.ts

```diff
@@ -10,3 +10,13 @@ export const fileFiltersSet: { label: string, filters: string[] }[] = [
 	{ label: localize('backup.filterBackupFiles', "Backup Files"), filters: ['*.bak', '*.trn', '*.log'] },
 	{ label: localize('backup.allFiles', "All Files"), filters: ['*'] }
 ];
+
+// Backup media device type
```

Comment:
Please add description here of what these values are and preferably provide a link to some external docs or something with the list of values these map to.